css_stylesheet.css

```diff
@@ -199,6 +203,45 @@ footer {
 	color: #343A40;
 }
 
+
+@media screen and (max-width: 1024px) and (min-width:770px) {
+	
+	.portfolioButtons {
+		width: 24%;
+	}		
+	
+	.paragraphs {
+		display:block
+	}
+	
+}
+
+@media screen and (max-width: 770px) {
+
+	.portfolioButtons {
+		width: auto;
+	}		
+		
+	.paragraphs {
+		display:block
+	}
+	
+	.portfolioButtons {
+		width: 100%;
+		flex-direction: column;
+	}
+		
+	.myName {
+		float: left;
+		margin-top: 0px;
+	}
+	
+	.pressHereAnchor {
+		font-weight: 900;
```

Comment:
Something that was picked up in my code review - changing font weight can result in things being nudged along the navigation menu. Might be worth testing whether items on your webpage change position when moused over.

Reply:
changed all instances to bold

Reply:
Not quite what I meant - changing the font weight at all can nudge the position of adjacent elements. If this isn't happening then there's nothing to worry about.

Reply:
My mistake. It's not an issue on mine but what would the solution be?